### PR TITLE
Removes classification loss

### LINKED ISSUE #9: Classification loss
@Adrien987k thank you for the work and the implementation.

I have noticed that you are also using [classification loss](https://github.com/facebookresearch/VICRegL/blob/main/main_vicregl.py#L530), in addition to the VICReg and VICRegL losses. The classification loss is not mentioned in the paper and was also not used in the original VICReg.

On the first hand, it seems a bit off, given that you won't have labels available when pretraining in the wild and it destroys the purpose of "self-supervised" pre-training ?

### PR DESCRIPTION
**Pull request by ds2268**
Closes #9

### CODE REVIEW

**Comment by ds2268** [comment]
An additional flag was needed for DDP to work, as there are now unused parameters, as the logits are not used in the loss. The optimal solution would completely throw out the logits from the forward pass output.

From the [DDP documentation](https://pytorch.org/docs/stable/notes/ddp.html):

``find_unused_parameters` is set to True. This mode allows running backward on a subgraph of the model, and DDP finds out which parameters are involved in the backward pass by traversing the autograd graph from the model output and marking all unused parameters as ready for reduction. During the backward pass, the Reducer would only wait for unready parameters, but it would still reduce all buckets. Marking a parameter gradient as ready does not help DDP skip buckets as for now, but it will prevent DDP from waiting for absent gradients forever during the backward pass. Note that traversing the autograd graph introduces extra overheads, so applications should only set find_unused_parameters to True when necessary.`
